packages/tql/README.md

```diff
@@ -0,0 +1,166 @@
+# What is it?
+
+```sh
+pnpm i @vercel/tql
+```
+
+TQL (Template-SQL -- unfortunately, T-SQL is already taken!) is a lightweight library for writing SQL in TypeScript:
```

Comment:
With the `@vercel/` prefix now, you can probably use `tsql` if you want! If not, we can remove this line.

Reply:
T-SQL is still taken (Transact-SQL, Microsoft's dialect used in SQL Server), but I removed the statement now that it's less obvious.